src/commands/fun/f.ts

```diff
@@ -1,16 +1,29 @@
-import { Message } from 'discord.js';
+import { ApplicationCommandOptionData, CommandInteraction, GuildMember, Message } from 'discord.js';
 import { Command } from '@lib/types/Command';
 
 export default class extends Command {
 
-	description = 'Press F to pay respects';
-	aliases = ['respect'];
+	description = 'Press F to pay respects.';
+	options: ApplicationCommandOptionData[] = [
+		{
+			name: 'target',
+			description: '(Optional) The user to pay respects to',

```

Comment:
I don't think this needs an `Optional` here, discord lets people know its optional

Reply:
I simple put that there in case the "+1 optional" on Discord wasn't clear enough (since it is small print), but shall remove prior to the merge.